### PR TITLE
Readme and Metadata updates

### PR DESCRIPTION
**Pull request by jonas-schievink**
(Includes #16)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,9 +1,13 @@
 [package]
 name = "r0"
-authors = ["Jorge Aparicio <japaricious@gmail.com>"]
-description = "Initialization code ('crt0') written in Rust"
+authors = [
```
Comment (thejpster) [inline Cargo.toml:3]: We agreed this would just be the team from now on.

**Review by thejpster — CHANGES_REQUESTED** [review]

**Comment by thejpster** [comment]
Hopefully we can squeeze this in before #17

**Review by almindor — APPROVED** [review]

**Review by thejpster — APPROVED** [review]

**Pull request merged**